File: BACKEND/DOTNETAPI/Solution1/UserLoginService/Models/Event.cs
using System;
using System.Collections.Generic;

namespace UserLoginService.Models;

public partial class Event
{
    public int EventId { get; set; }

    public string? EventName { get; set; }

    public string? Description { get; set; }

    public DateOnly? Date { get; set; }

    public double? Price { get; set; }

    public string? Address { get; set; }

    public string? City { get; set; }

    public long? Pincode { get; set; }

    public int? NoOfSeats { get; set; }

    public int? Catid { get; set; }

    public string? Status { get; set; }

    public virtual ICollection<Booking> Bookings { get; set; } = new List<Booking>();

    public virtual Category? Cat { get; set; }

    public virtual ICollection<Feedback> Feedbacks { get; set; } = new List<Feedback>();
}

File: BACKEND/SpringBootAPI/src/main/java/com/example/demo/entities/Event.java
package com.example.demo.entities;

import java.sql.Date;
import java.time.LocalDate;

import com.fasterxml.jackson.annotation.JsonIgnore;

import jakarta.persistence.*;
import lombok.*;

@Entity
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@ToString
@Table(name = "event")

public class Event {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "event_id")
    private int eventId;

    @Column(name = "event_name")
    private String eventName;

    @Column(name = "description")
    private String description;

    
    @Column(name = "date")
    private Date date;

    @Column(name = "price")
    private double price;

    @Column(name = "address")
    private String address;

    @Column(name = "city")
    private String city;

    @Column(name = "pincode")
    private long pincode;

    @Column(name = "no_of_seats")
    private int noOfSeats;

//    @ManyToOne(fetch = FetchType.EAGER)
//    @JoinColumn(name = "userid") 
//    @JsonIgnore
//    private User user;
    @Column(name = "status")
    private String status;

    @ManyToOne(fetch = FetchType.EAGER)
    @JoinColumn(name = "catid") 
    private Category category;

  
}
